test/track.py

```diff
@@ -11,9 +11,11 @@ class Track:
     tags = attr.ib(factory=list)
 
     @classmethod
-    def create(cls, recording):
+    def create(cls, recording, start_s=10):
         "Make a TestTrack with some plausible data."
-        return cls(None, recording, data={"foo": [[1, 2], [3, 4]], "start_s": 10, "end_s": 20})
+        if start_s is None:
```

Comment:
Why have this and also have the parameter default be 10?  Genuine question, I'm a python noob, and there may be a good reason?

Reply:
No good reason, left overs from dev